backend/app.py

```
@@ -309,11 +258,17 @@ def analyze():
                         elif threat_score >= 50: severity = "HIGH"
                         elif threat_score >= 25: severity = "MEDIUM"
                         
+                        # Chain-of-custody ledger: append this evidence intake
+                        # record (filename + SHA-256 + timestamp) to the case.
+                        custody_chain = case.get("custody", [])
+                        custody_chain = [custody] + custody_chain
```

Ruff (0.16.2)

[warning] Consider `[custody, *custody_chain]` instead of concatenation

Replace with `[custody, *custody_chain]`

(RUF005)